contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/SavedSearchesModal.vue

```diff
@@ -189,7 +176,44 @@
 
 <style scoped lang="scss">
 
+  .empty-state {
+    padding: 8px;
```

Comment:
Would you remove this padding? I realize that you used it to preserve the original experience as requested - thanks for paying attention to that :)

I just think that here it'd be more meaningful to rely on default `KModal` alignment (even though in some cases we make exceptions, generally one of the purposes of KDS components in consistency).